pkg/devicetemplates/microchip.go

```diff
@@ -33,8 +33,20 @@ import (
 	jose "gopkg.in/square/go-jose.v2"
 )
 
-var microchipPublicKeys = map[string][]byte{
-	"B0uaDLLiyKe-SorP71F3BNoVrfY": []byte(`-----BEGIN CERTIFICATE-----
+var microchipPublicKeys = map[string]map[string][]byte{
+	"ATECC608A-MAHTN-T": {
```

Comment:
Maybe add the part number as a constant ? It's mentioned also in the `init` and it's not a simple constant.